x/concentrated-liquidity/grpc_query.go

```diff
@@ -102,3 +102,30 @@ func (q Querier) Params(goCtx context.Context, req *types.QueryParamsRequest) (*
 	ctx := sdk.UnwrapSDKContext(goCtx)
 	return &types.QueryParamsResponse{Params: q.Keeper.GetParams(ctx)}, nil
 }
+
+// LiquidityDepthsForRange returns liquidity depths for the given range.
+func (q Querier) LiquidityDepthsForRange(goCtx context.Context, req *types.QueryLiquidityDepthsForRangeRequest) (*types.QueryLiquidityDepthsForRangeResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	maxRange := sdk.NewInt(10000)
```

Comment:
if we decide to internally hard cap this I think it should live in a const somewhere

Reply:
defined a constant at the top of `grpc_query.go`, lmk how it works out

Reply:
Oh I had meant constants.go https://github.com/osmosis-labs/osmosis/blob/d28c7412c8d16ed48a20f7b770185d238d83f12b/x/concentrated-liquidity/types/constants.go#L1